tools/clang/unittests/HLSLExec/ShaderOpArith.xml

```diff
@@ -3776,10 +3776,21 @@ void MSMain(uint GID : SV_GroupIndex,
         }
         #endif
 
+
+        #ifdef FUNC_TEST_CAST
+        vector<OUT_TYPE, NUM> TestCast(vector<TYPE, NUM> Vector)
+        {
+          return (vector<OUT_TYPE, NUM>)Vector;
+        }
+        #endif
+
+        #ifdef FUNC_TERNARY_ASSIGNMENT
+        vector<TYPE, NUM> TestTernaryAssignment(vector<TYPE, NUM> Vector,
+        vector<TYPE, NUM> Vector2))
```

Comment:
Seems suspicious to me that this change is adding an `#ifdef` but isn't also adding a corresponding `#endif`.

Reply:
Bad merge on my part. Fixed.